docker-compose.yml

```diff
@@ -3,7 +3,10 @@ version: '3'
 services:
   web:
     container_name: proxitok-web
-    image: ghcr.io/pablouser1/proxitok:master
```

Comment:
You should let docker-compose use the pre-built image, maybe you were doing some testing and forgot to undo it?

Reply:
Obviously, how else could I test? It's just Github automatically picking up the `docker-compose` changes, this wasn't meant to make it in the PR